WooCommerce/Classes/Authentication/AuthenticatedWebViewModel.swift

```diff
@@ -10,6 +11,10 @@ protocol AuthenticatedWebViewModel {
     /// Initial URL to be loaded on the web view
     var initialURL: URL? { get }
 
+    /// Optional credentials for authenticating with WP.org
+    ///
+    var wporgCredentials: WordPressOrgCredentials? { get }
```

Comment:
Good point! I thought this will be needed for the Woo installation flow, but then we don't need it since the user should be authenticated with WP.com at that point. I moved `wporgCredentials` to `AuthenticatedWebViewController` instead in eef45f41e14881031f21d2a0ab82e4e60fda6b63 and 1863acda314d092d7c78357cd4750b9a203cecec.